### ISSUE TITLE
Suggestion : Implement Fog Of War.

### ISSUE THREAD

**Issue by bchoii** [SpaceTradersAPI/issues-and-suggestions#51]
Currently you can see the whole system map just by calling the https://api.spacetraders.io/game/systems endpoint.
This could be made more interesting.

Suggest to 
1. remove this endpoint. 
2. only reveal your ships' planets, and planets near it / planets that can travel from it.
(imagine a wormhole that is accessible from only a certain planet to a distant galaxy)

Purpose
Extend gameplay mechanics. eg map making and one way routes.

**Comment by bchoii** [comment]
It might not be beginner friendly ? 

Your beginner planet can still reveal all planets. Just under the ship / planet endpoint rather than a systems endpoint.
Beginners can still play it as it currently is.

This limitation does not need to apply to the whole galaxy, maybe only at the edges or certain obscure planet..

**Comment by artokun** [comment]
This is a great idea. It will give us more flexlibility for creating a more discoverable universe that gets mapped out by the community.

I approve the following:
1. Removing this endpoint
2. Show planets within 1-way fuel range radius from the ship doing the navigationScan
3. Returns out of range objects as anomalies up to 2x your 1-way fuel range.

This might also apply to Goods

**Comment by thebrubaker** [comment]
I agree, adding a concept of explorability would be amazing. You can imagine if you have ship 123 at planet OE-A1 and you hit a `/ships/123/scan` endpoint it returns all locations within your ships scan range. Some interesting locations could be way out of range until upgrading or reaching top tier ships.

**Comment by bchoii** [comment]
Possible gameplay...
1. Far away planets.
2. One way wormholes, that open up only on certain date / times.
3. Asteriods that can be near, but too small to detect unless you have a certain upgrade.
